packages/fiori-freestyle-writer/test/common.ts

```diff
@@ -9,10 +10,16 @@ export const testOutputDir = join(__dirname, '/test-output');
 
 export const debug = prepareDebug();
 
+/**
+ * @returns object
+ *          object.enabled debug enabled boolean
+ *          object.outputDir output directory
+ */
 export function prepareDebug(): { enabled: boolean; outputDir: string } {
     const debug = !!process.env['UX_DEBUG'];
-    // eslint-disable-next-line no-console
+
     if (debug) {
+        // eslint-disable-next-line no-console
```

Comment:
Perhaps we can agree, that usage of `console` is ok for `test` folder or avoid it everywhere.

Reply:
Good catch. Actually already allowed now in test folder rules. 
Remove disabled `no-console` in https://github.com/SAP/open-ux-tools/pull/689/commits/582c871523276087e806a7b80ddbb88a81111a27